src/apps/cacheStatus/options.cpp

```diff
@@ -222,17 +222,16 @@ void COptions::Init(void) {
     mode = "";
 
 #ifndef HOST_NAME_MAX
-#define HOST_NAME_MAX 64
+#define HOST_NAME_MAX 256
+#endif
+#ifndef LOGIN_NAME_MAX
 #define LOGIN_NAME_MAX 64
 #endif
     char hostname[HOST_NAME_MAX];
```

Comment:
Might want to do go with `char hostname[HOST_NAME_MAX+1] = { 0 };` here:
```
gethostname() returns the null-terminated hostname in the character array name, which has a length of len bytes.  If the null-terminated hostname is too large to fit, then the name is truncated, and no error is returned (but see NOTES below).  POSIX.1 says that if such truncation occurs, then it is unspecified whether the returned buffer includes a terminating null byte.
```

Reply:
Read my proposal again. I am suggesting making the variable bigger not assigning to the n+1'th element.